src/main/java/org/asamk/signal/commands/Commands.java

```diff
@@ -11,6 +11,7 @@
         addCommand("addDevice", new AddDeviceCommand());
         addCommand("block", new BlockCommand());
         addCommand("daemon", new DaemonCommand());
+        addCommand("isRegistered", new IsRegisteredCommand());
```

Comment:
hm, maybe `getRegistrationStatus` or `getUserStatus` would be more flexible in case we add more information in the future.